src/renderers/webgl/WebGLExtensions.js

```diff
@@ -12,26 +33,23 @@ function WebGLExtensions( gl ) {
 
 		let extension;
 
-		switch ( name ) {
+		if ( aliasses[ name ] === undefined ) {
```

Comment:
TBH, this feels a bit over-engineered to me. Yes, the previous code was dumper but also easier to read. I tend to leave the code as it is.